korim/src/commonMain/kotlin/com/soywiz/korim/format/ASE.kt

```diff
@@ -564,7 +572,15 @@ object ASE : ImageFormatWithContainer("ase") {
                     maskRotate = resolved.bitmask90CWFlip,
                 )
             }
-            return ImageFrameLayer(layer, resolved.bmp.slice(), resolved.x, resolved.y, main = false, includeInAtlas = true, tilemap = tilemap)
+            return ImageFrameLayer(
```

Comment:
This looks like a reformat. Should it be included in this PR?

Reply:
I just auto format with the IDE bot worth creating separate PR imo